lib/model/autocomplete.dart

```diff
@@ -243,11 +250,22 @@ class MentionAutocompleteView extends ChangeNotifier {
     }
   }
 
-  Iterable<MentionAutocompleteResult> get results => _results;
-  List<MentionAutocompleteResult> _results = [];
+  Iterable<R> get results => _results;
+  List<R> _results = [];
+
+  Future<void> _startSearch(Q query) async {
```

Comment:
The implementation of this method is changed in #693, specifically, there is no retry now. PTAL.

Reply:
Oh sorry! I must have missed this one while rebasing.